test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/GeneratorTests.cs

```diff
@@ -85,6 +86,49 @@ public void GenerateJsonDocumentTest_Package()
         Assert.AreEqual(expectedJsonContentAsString, generatedJsonString);
     }
 
+    [TestMethod]
+    public void GenerateJsonDocument_DependsOnId_Null_ReturnsNull()
+    {
+        var packageInfo = new SbomPackage
+        {
+            PackageName = "TestPackage",
+            DependOn = null
+        };
+
+        var result = generator.GenerateJsonDocument(packageInfo);
+
+        Assert.IsNull(result.ResultMetadata.DependOn, "DependOnId should be null when DependOn is null.");
```

Comment:
You probably don't need the description in lines 100, 114, and 129. These tests are single case and will already report the expected and actual values.